### PR TITLE
Fix mt_rand return type

### PR DESCRIPTION
**Pull request by enumag**
Closes https://github.com/phpstan/phpstan/issues/8272

### CODE REVIEW

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]
What did you do to the icon? 👓

**Comment by enumag** [comment]
Huh? Nothing at all. I didn't touch it.

**Comment by enumag** [comment]
It must have somehow been deleted by git automatically because I have .idea in global gitignore... I'll add it back...

**Comment by ondrejmirtes** [comment]
You deleted the file.

**Comment by enumag** [comment]
Fixed.

File: tests/PHPStan/Analyser/data/bug-8272.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+use function PHPStan\Testing\assertType;
+
+assertType('int<1, 5>', mt_rand(1, 5));
```
Comment (staabm) [inline tests/PHPStan/Analyser/data/bug-8272.php]: This test file is not included from any test-class?

Should be yielded from NodeScopeResolverTest
(And the line wrapped in a method or function)

Comment (ondrejmirtes) [reply tests/PHPStan/Analyser/data/bug-8272.php]: And in a namespace.

Comment (canvural) [reply tests/PHPStan/Analyser/data/bug-8272.php]: And maybe tested with no arguments.

**Comment by ondrejmirtes** [comment]
I can't review it when the tests are failing.

**Comment by enumag** [comment]
@ondrejmirtes Fixed. There is still one failed job but I don't think it's related to my changes: https://github.com/phpstan/phpstan-src/actions/runs/3393592592/jobs/5641134975

**Comment by ondrejmirtes** [comment]
Thank you.

**Pull request merged**